mmv1/templates/terraform/examples/uptime_check_config_cloud_run.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_monitoring_uptime_check_config" "<%= ctx[:primary_resource_id] %>" {
+  display_name = "<%= ctx[:vars]["display_name"] %>"
+  timeout      = "60s"
+  user_labels  = {
+    example-key = "example-value"
+  }
+
+  http_check {
+    path = "some-path"
+    port = "443"
+  }
+
+  monitored_resource {
+    type = "cloud_run_revision"
+    labels = {
+      location     = google_cloud_run_v2_service.<%= ctx[:vars]["cloud_run_primary_id"] %>.location
```

Comment:
Were you able to run this test locally? I'm asking because it doesn't look to me like it will work, because these references to a `google_cloud_run_v2_service` are missing. You probably have two options: hardcode dummy values for these labels, or create a cloud run service as part of this test.

Reply:
Ah, sorry, I added in this commit https://github.com/GoogleCloudPlatform/magic-modules/pull/10820/commits/53e20a8ffa1e831b3a70dbe9eaa621db0cdf8434

Thank you for pointing it out!